scripts/h5toobj.py

```diff
@@ -290,42 +225,45 @@ def dask_from_chunks(chunks, out_name, base_name):
 
     use_rados = args.ceph_pool is not None
     if use_rados:
-        store = open_rados(args.ceph_conf, args.ceph_pool)
+        obj_store = RadosChunkStore(args.ceph_conf, args.ceph_pool,
+                                    args.ceph_keyring)
+        pool_stats = obj_store.ioctx.get_stats()
+        logger.info("Connected to pool %s. Currently holds %d objects "
+                    "totalling %g GB", args.ceph_pool,
```

Comment:
GB -> GiB if you're using 2**30.

Reply:
Simon's code - missed that...

Reply:
Stuck with GB and divided by 1e9 instead.